x11-misc/yabar/Manifest

```diff
@@ -0,0 +1 @@
+DIST yabar-a0d3fdfed992149b741eb8fcf53f02b5d1a6142e.tar.gz 892725 BLAKE2B eb14b06327533034756885c68ab03b1ff049c987d1dac6d083c553646cd178210429d9457c1067db3211fcb39e9814a260aed201dea4e6410544b9247c5dee93 SHA512 f93250605c5b56d64c7ec6a7e7faa6120cdc06191ddafb13822875fd98b1b7a1822d9b93b62c49c302b1d4bffb14a200feed937efb0b11711de7159ce83eb7c5
```

Comment:
This version isn't really recognizable. I see upstream has tagged releases, so use something like like `0.4.0_p20190720` in here instead please.

Reply:
changes made to reflect tagged release in manifest